c2corg_api/models/document.py

```diff
@@ -53,7 +54,8 @@ def redirects_to(self):
             Integer, ForeignKey(schema + '.documents.document_id'),
             nullable=True)
 
-    quality = Column(enums.quality_type)
+    quality = Column(
+        enums.quality_type, nullable=False, server_default=quality_types[1])
```

Comment:
Yes, the client told me that the default value for migrated documents should be "medium" and "stub" for new documents (unless the contributor changes the quality value when saving the document - which is the encouraged behaviour).